torch/_dynamo/variables/builder.py

```diff
@@ -1585,9 +1591,20 @@ def _clone_input(value):
     elif isinstance(example_value, bool) and proxy.node.target in [
         torch.backends.cuda.can_use_flash_attention,
         torch.backends.cuda.can_use_efficient_attention,
+        torch.backends.cuda.flash_sdp_enabled,
+        torch.backends.cuda.mem_efficient_sdp_enabled,
+        torch.backends.cuda.math_sdp_enabled,
     ]:
         proxy.node.meta["example_value"] = example_value
         return ConstantVariable.create(example_value, **options)
+    elif isinstance(example_value, float) and proxy.node.target in [
+        torch.sym_sqrt,
+    ]:
+        proxy.node.meta["example_value"] = example_value
+        return ConstantVariable.create(example_value, **options)
```

Comment:
uhhh you sure???

Reply:
- Oh right, we want to constant fold this and never reach this function (as per the comment above the function!)
- I'm going to try to use normal `math.sqrt` instead of sym_sqrt and see what happens. Apparently there is a comment there about normal sqrt not working, but I'm not sure why yet.